winvnc/winvnc/VirtualDisplay.cpp

```diff
@@ -445,11 +470,7 @@ bool VirtualDisplay::InstallDriver(bool fromCommandline)
 			if (p == NULL)
 				return 0;
 			*p = '\0';
-#ifdef _X64
-			strcat_s(szdriverPath, "\\UVncVirtualDisplay64\\UVncVirtualDisplay.inf");
-#else
 			strcat_s(szdriverPath, "\\UVncVirtualDisplay\\UVncVirtualDisplay.inf");

```

Comment:
Perhaps we need to call the x64 files UVncVirtualDisplay64.inf and place then in a folder UVncVirtualDisplay
But not the same location and name for 64 and 32-bit